### DISCUSSION TITLE
IP Address looks banned

### DISCUSSION THREAD

**Discussion by HoChanHwang** [vercel/community#6099] [Help]
### Summary

I made a lot of request to my vercel deployed page and suddenly can't access to page in same ip address but can access from other address.

### Steps to Reproduce

1. deploy vercel preview
2. make lot of request to page
3. can't access to page

share my code that make lot of request. I repeated 3~5 times.
```
Array.from({length:10000}, ((_, index) => {
    fetch('https://my-preview-url.com').then(() => console.log(index));
}));
```

**Comment by Worrah** [comment]
did you resolve it eventually? seems like i've got the same problem

**Reply by HoChanHwang** [reply]
I didn't do anything but, it solved after 1~2 weeks later
